scss/_section.scss

```diff
@@ -0,0 +1,29 @@
+.section {
+    .border-wrapper {
+        padding: 15px;
+        background-color: rgba(#000,0.4);
+        border-radius: 3px;
+    }
+    .sign-content-wrapper,
+    .content-wrapper {
+        background-color: $backgroundWhite;
+        padding: 25px;
+        border-radius: 10px;
+    }
+    
+    .section-heading {
+        text-align: center;
+        color: #fff;
+        font-size: 34px;
+        margin-top: 50px;
```

Comment:
margin top i bottom o jedne wartosci mozesz zapisać jakos margin: 50px 0 jeżeli nie nadpisze to innych dziedziczonych marginów.